### PR TITLE
Codewind Imported Projects Restructure

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

For Issue 1762: https://github.com/eclipse/codewind/issues/1762

@micgibso and/or @sishida, please review and assign me. 

Note, this doc deals with several of the topics for our style guide that we put on hold. Such as headings.

### CODE REVIEW

File: docs/_documentations/importing-existing-projects.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ MicroProfile projects are Java applications that are deployed to WebSphere Liber
...
 
-Avoid copying files from the project’s Maven target folder as part of any Dockerfile instructions because Codewind builds your project within the container. The application builds against the same runtime that is used in production to avoid inconsistencies between development and production environments.
+Avoid copying files from the project’s Maven target folder to any Dockerfile instructions because Codewind builds your project within the container. The application builds against the same runtime that is used in production to avoid inconsistencies between development and production environments.
 
 ### Pre-import instructions
 
 MicroProfile projects must be configured to build by using Maven.
 
-Tip: For an example of a working MicroProfile application, create a new MicroProfile project in Codewind. You can see the `pom.xml` file contents and use the file as a template when you configure your project's `pom.xml` file.
+**Note:** for an example of a working MicroProfile application, create a new MicroProfile project in Codewind. You can see the `pom.xml` file contents and use the file as a template when you configure your project's `pom.xml` file.
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: After terms like "Note:", begin the sentence with a capital letter. In this instance, "For" needs capitalization.

File: docs/_documentations/importing-existing-projects.md [hunk 2]
```diff
@@ -93,30 +93,30 @@ Configure your `pom.xml` file as follows:
   </profile>
   ```
 
-  Then add the required Liberty Maven plugin configuration:
+3. Add the required Liberty Maven plugin configuration:
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: "Plug-in" should have the hyphen.

File: docs/_documentations/importing-existing-projects.md [hunk 3]
```diff
@@ -93,30 +93,30 @@ Configure your `pom.xml` file as follows:
...
 
-  2b. Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
+- Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: Maybe... "Add the `server.xml` Liberty server configuration file that is located..."

File: docs/_documentations/importing-existing-projects.md [hunk 4]
```diff
@@ -93,30 +93,30 @@ Configure your `pom.xml` file as follows:
...
 
   ```xml
   <configFile>${basedir}/src/main/liberty/config/server.xml</configFile>
   ```
 
-  2c. Optional: Liberty server environment file:
+- Optional: Liberty server environment file:
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: In line 104 and in line 110, I think users will understand that "Add" is the implied verb here. For added clarity, though, I'd go ahead and include the verb "Add."
- Add the Liberty server...
- Add the Liberty Java...

File: docs/_documentations/importing-existing-projects.md [hunk 5]
```diff
@@ -164,10 +164,10 @@ FROM ibmjava:8-sfj
...
 
-- Ensure the project provides a `start` npm script in `package.json` so it can be started by `npm start`.
-    - For example: `start: "node server/server.js"`
-- In local installations of Codewind, you can restart the project in Debug mode. To use this feature, the project must also provide a `debug` npm script, which accepts connections from all hosts on port 9229. For help configuring this script, see [the Node.js debugging guide](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options).
-    - For example: `debug: "node --inspect=0.0.0.0:9229 server/server.js"`
+- Ensure the project provides a `start` npm script in `package.json`, so it can be started by `npm start`.
+    - For example: `start: "node server/server.js"`.
+- In local installations of Codewind, you can restart the project in Debug mode. To use this feature, the project must also provide a `debug` npm script which accepts connections from all hosts on port 9229. For help configuring this script, see [the Node.js debugging guide](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options).
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: Comma needed before "which" to set off grammatically nonessential clause. Or, and this options is probably even better, replace "which" with "that."

File: docs/_documentations/importing-existing-projects.md [hunk 6]
```diff
@@ -164,10 +164,10 @@ FROM ibmjava:8-sfj
...
+    - For example: `debug: "node --inspect=0.0.0.0:9229 server/server.js"`.
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: Line 170 isn't conveying a complete sentence, so the period is not necessary. If the code phrase was part of a complete sentence, though, we would want to keep the period. :)

File: docs/_documentations/importing-existing-projects.md [hunk 7]
```diff
@@ -176,7 +176,7 @@ If you have a `Dockerfile`, it must meet the following requirements:
     - For example, if you're using port 3000, the `Dockerfile` needs to include `EXPOSE 3000`.
 - Ensure that the application is located in the `/app` directory within the Docker container.
 - Ensure that the `Dockerfile` sets the working directory to `/app`:
-    - `WORKDIR "/app"`
+    - `WORKDIR "/app"`.
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: How about...`Ensure that the `Dockerfile` sets the working directory to `/app` with `WORKDIR "/app"`.`

File: docs/_documentations/importing-existing-projects.md [hunk 8]
```diff
@@ -186,13 +186,13 @@ Codewind works with Swift projects that use the Kitura web framework.
 Requirements:
 
 - A `Dockerfile-tools` file is generated to build the project. Ensure the project can be built by using a `release` build configuration.
-For example, you should be able to build the project by using the command
+For example, you should be able to build the project by using the command:
```
Comment (sishida) [inline docs/_documentations/importing-existing-projects.md]: "Should" indicates uncertainty. Unless we know this command is unstable (in which case we'd want to open an issue), avoid "should." Instead go for something like, "For example, you can build the project by using the following command:"

**Comment by j-c-berger** [comment]
@sishida, made your suggested changes in the new commit, thanks!

**Pull request merged**